docs/node-groups.md

```diff
@@ -156,12 +161,15 @@ The below example demonstrates advanced configuration options using Spot/GPU ins
       min_size     = 2
 
       # Node Group update configuration. Set the maximum number or percentage of unavailable nodes to be tolerated during the node group version update.
-      max_unavailable = 1 # or percentage = 20
+      update_config = [{
+        max_unavailable            = 1
+      }]
 
       # Node Group compute configuration
-      ami_type       = "AL2_x86_64"
-      capacity_type  = "SPOT"
-      instance_types = ["t3.medium", "t3a.medium"]
+      ami_type               = "AL2_x86_64"
+      capacity_type          = "SPOT"
+      instance_types         = ["r5d.xlarge", "r5a.xlarge"]
```

Comment:
So we don't throw off users since this requires the `AL2_ARM_64` AMI type
```suggestion
      instance_types         = ["r5d.xlarge", "r5.xlarge"]
```

Reply:
In this case i am looking for instance type with NVMe disks to show the formatting. I will update this example with correct instance types